src/main/app/src/app/gebruikersvoorkeuren/zoekopdracht-save-dialog/zoekopdracht-save-dialog.component.less

```diff
@@ -0,0 +1,5 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
```

Comment:
Laten we een lege less file nou staan? Ik zou zeggen dat ie weg mag.

Reply:
Op de meeste plekken staan ze gewoon, dus laat ze voor nu staan